src/pperm.c

```diff
@@ -288,28 +288,10 @@ UInt RANK_PPERM4(Obj f)
 
 static Obj SORT_PLIST_CYC(Obj res)
```

Comment:
Not your fault, but this function is really named incorrectly. It should be called `SORT_PLIST_INTOBJ`. I can take care of that in a later PR (but of course if you are willing to do it here, you are more than welcome ;-)

Reply:
Alternatively, just fold it into the calling code (but again, I can take care of that in a later PR)